networking/src/p2p/peer.rs

```diff
@@ -315,6 +541,22 @@ impl Receive<SendMessage> for Peer {
     type Msg = PeerMsg;
 
     fn receive(&mut self, ctx: &Context<Self::Msg>, msg: SendMessage, _sender: Sender) {
+        match self.throttle_quota.lock() {
+            Ok(ref mut quota) => {
+                if let Err(quota) = quota.can_send(msg.message.as_ref()) {
+                    if quota == -1 {
```

Comment:
@akoptelov 
also here, just one IF `if let Err(quota) =`  should be enought

Reply:
See above.